### DISCUSSION TITLE
Service "keycloak/keycloak" does not have any active Endpoint.

### DISCUSSION THREAD

**Discussion by JairAguirre** [keycloak/keycloak#31389] [Q&A]
Hello team, I have deployed Keycloack in kuernetes cluster using bitnami charts and My pods for keycloak are running normally 

![image](https://github.com/user-attachments/assets/cfbb4980-6b6a-416c-94de-22eb01f8acee)

Logs for keycloak pod: 

![image](https://github.com/user-attachments/assets/71a44cb4-3a9f-45d9-be57-a507b64b8873)

Logs for keycloak postgresdb: 
![image](https://github.com/user-attachments/assets/b951766c-22d5-4c6a-afe7-e82afd0f06d8)

But when I'm  trying to access to Keycloak from browser the page never load, and instead  I'm getting on my ingress engine: 
Service "keycloak/keycloak" does not have any active Endpoint.

![image](https://github.com/user-attachments/assets/c54af3c2-ef2e-43a7-9ac7-73f972cabc7a)

May someone give me a hint to debug this error?

**Comment by alaa-ledger** [comment]
did you ever get around to fixing this? I'm running into the same thing

**Comment by shawkins** [comment]
@alaa-ledger @JairAguirre I'm not familiar with what the bitnami chart is doing under the covers in setting up the ingress. The keycloak operator for example creates a Service to access the Pods, which is then referenced by an Ingress. As long as the Pods are ready, they are accessible via the Service. 

So you'll need to look more deeply at how the bitnami ingress and any underlying Kubernetes Services are defined to see why the Pods are not seen as available.
